EMIP/1001-2000/EMIP01909.xml

```diff
@@ -32,37 +32,37 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                   <idno facs="EMIP/Codices/1909/">Hazen Codex 5</idno>
                </msIdentifier>
                <msContents>
-                  <summary>Image of the Covenant of Mercy. Image of Saint Mark. Image of Gabriel. Image of Gäbrä Mänfas Qǝddus. Image of Peter and Paul. Image of Rufa’e፡. Image of the Savior of the World. Layman's Prayer</summary>
+                  <summary>Image of the Covenant of Mercy; Image of Saint Mark; Image of Gabriel; Image of Gabra Manfas Qǝddus; Image of Peter and Paul; Image of Rufāʾel; Image of the Savior of the World; Layman's Prayer</summary>
```

Comment:
please do not waste time on polishing EMIP records unless there are obvious XML or content errors, there are over 6000 of very dirty files which are not our priority (I only go through them to reassign to true repositories and to match to already existing records to avoid doublets in Bm)